haskell/package.yaml

```diff
@@ -13,13 +13,7 @@ extra-doc-files:
 - README.md
 
 extra-source-files:
-- test/segmented_cfmm_FA12.tz
-- test/segmented_cfmm_FA12_CTEZ.tz
-- test/segmented_cfmm_FA12_FA2.tz
-- test/segmented_cfmm_FA2.tz
-- test/segmented_cfmm_FA2_CTEZ.tz
-- test/segmented_cfmm_FA2_FA12.tz
-- test/storage_default.tz
+- test/*.tz
```

Comment:
Ah, sadly this doesn't quite work because AFAIU when CI checks the cabal files the `.tz` sources aren't there (so it will produce `.cabal`s with different `extra-source-files` than what we have locally.

Perhaps it can be fixed, but the last time I touched this I left the list explicitely.